### ISSUE TITLE
Python 3.5 is not actually supported

### ISSUE THREAD

**Issue by rominf** [LuminosoInsight/ordered-set#67]
I'm building ConceptNet 5.7 on Ubuntu 16.04 with Python 3.5. I get following error:
```
  File "/home/conceptnet/conceptnet5/conceptnet5/db/prepare_data.py", line 6, in <module>
    from ordered_set import OrderedSet
  File "/home/conceptnet/env/lib/python3.5/site-packages/ordered_set.py", line 51, in <module>
    class OrderedSet(MutableSet[T], Sequence[T]):
  File "/home/conceptnet/env/lib/python3.5/site-packages/ordered_set.py", line 186, in OrderedSet
    def update(self, sequence: SetLike[T]) -> int:
  File "/usr/lib/python3.5/typing.py", line 546, in __getitem__
    "Cannot subscript an existing Union. Use Union[u, t] instead.")
TypeError: Cannot subscript an existing Union. Use Union[u, t] instead.
```

Looks like compatibility with Python 3.5 was lost. I suggest to remove it from a list of supported Python versions.

**Comment by rspeer** [comment]
oof. I can just go fix Python 3.5 support -- that'll be better.

**Comment by rspeer** [comment]
Fixed in 4.0.2.
